wsdl/ver20/media/wsdl/media.wsdl

```diff
@@ -179,12 +179,12 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 			<!--===============================-->
 			<xs:simpleType name="SupportedAudioClipFormat">
 				<xs:restriction base="xs:string">
-					<xs:enumeration value="WAV"/>
-					<xs:enumeration value="MP3"/>
-					<xs:enumeration value="OGG-Opus"/>
-					<xs:enumeration value="OGG-Vorbis"/>
-					<xs:enumeration value="OGG-Flac"/>
-					<xs:enumeration value="OGG-Speex"/>
+					<xs:enumeration value="audio/wav;codecs=g711-ulaw"/>
```

Comment:
I assumed "WAV" would correspond to "audio/wav;codecs=lpcm"
Perhaps we should add that one as well?
There is also the old  "audio/basic" (.au files) with g711-ulaw that may be worth mentioning?
Should the description of "Format of the audioclip. Supported audio Clip formats are .." be rephrased to:
"Supported audio Clip formats includes ..." to no make it sound as a limit?

Reply:
@johado , mixing user defined string enumerations with iana template with codec strings will confuse the people. it is better to limit only with user defined string enumerations. i.e WAV-PCM, WAV-G711-ULAW. Format of the audio clip description, I will modify as per your suggestion.